### DISCUSSION TITLE
Not clearing the screen at each frame

### DISCUSSION THREAD

**Discussion by jaese** [aduros/wasm4#193] [Questions] [answered]
I want to implement a pattern of updating only when/where it needs to be updated on the screen and save some computation, but it seems Wasm4 clears the screen at the start of each frame so the whole screen needs to be redrawn.
Is there a way to configure this, or a reason for this design?

**Comment by DenialAdams** [comment, marked as answer]
There is a system flag to change this behavior, see SYSTEM_PRESERVE_FRAMEBUFFER

https://wasm4.org/docs/reference/memory

**Reply by jaese** [reply]
I missed that. Thanks for prompt answer!

**Reply by DenialAdams** [reply]
No problem :)
